index.bs

```diff
@@ -4295,28 +4295,41 @@ An [=interface=] can be declared to be asynchronously iterable by using an
 (matching <emu-nt><a href="#prod-AsyncIterable">AsyncIterable</a></emu-nt>) in the body of the
 [=interface=].
 
-<pre highlight="webidl" class="syntax">
+<!-- TODO: add highlight="idl" after idlparser gets updated -->
+<pre class="syntax">
     interface interface_identifier {
+      async iterable&lt;value_type&gt;;
+      async iterable&lt;value_type&gt;(/* arguments... */);
       async iterable&lt;key_type, value_type&gt;;
     };
 </pre>
 
 Objects that [=implement=] an [=interface=] that is declared to be asynchronously iterable support
 being iterated over asynchronously to obtain a sequence of values.
 
+If a single type parameter is given, then the interface has a
+<dfn export>value asynchronously iterable declaration</dfn> and asynchronously provides values of the
+specified type. If two type parameters are given, then the interface has a
+<dfn export>pair asynchronously iterable declaration</dfn> and asynchronously provides
+[=value pairs=] with the given types.
+
 Note: In the ECMAScript language binding, an interface that is asynchronously iterable will have
-<code class="idl">entries</code>, <code class="idl">keys</code>, <code class="idl">values</code>,
-and {{@@asyncIterator}} properties on its [=interface prototype object=].
+{{@@asyncIterator}} and <code class="idl">values</code> properties on its
```

Comment:
perhaps I'm just not familiar enough with the subject, but it would help me understand things better if some note explained that the optional arguments for the async iterable definition end up as arguments you can pass to the values method.

Reply:
Great point. I hooked up all the normative algorithms but did not write appropriate descriptive text. Will do.